images/iscsi-target-tgtd/iscsi-demo-target-tgtd.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
Will do this in a separate step - following the lines of: First add, then integrate.